lib/segment/src/types.rs

```diff
@@ -830,22 +899,21 @@ mod tests {
             _ => panic!(),
         }
 
-        let float_query_strict = r#"{"type": "float", "value": [1.0, 2.0, 3.0]}"#;
-        let val_float_query_strict: serde_json::value::Value =
-            serde_json::from_str(float_query_strict).unwrap();
-        let payload_interface_float_query_strict: PayloadInterface =
-            serde_json::from_value(val_float_query_strict).unwrap();
-        let payload_float_query_strict: PayloadType =
-            (&payload_interface_float_query_strict).into();
-        match &payload_float_query_strict {
-            PayloadType::Float(x) => {
-                assert_eq!(x.len(), 3);
-                assert_eq!(x[0], 1.0);
-                assert_eq!(x[1], 2.0);
-                assert_eq!(x[2], 3.0);
-            }
-            _ => panic!(),
-        }
+        // strict no longer supported.
+        //
+        // let float_query_strict =
+        //     r#"{"float_query_strict":{"type": "float", "value": [1.0, 2.0, 3.0]}}"#;
+        // let payload: Payload = serde_json::from_str(float_query_strict).unwrap();
+        // let payload_float_query_strict = payload.get("float_query_strict").unwrap();
+        // match &payload_float_query_strict {
+        //     PayloadType::Float(x) => {
+        //         assert_eq!(x.len(), 3);
+        //         assert_eq!(x[0], 1.0);
+        //         assert_eq!(x[1], 2.0);
+        //         assert_eq!(x[2], 3.0);
+        //     }
+        //     _ => panic!(),
+        // }
```

Comment:
To my mind it is better to just remove code, instead of commenting:
- We can always retrieve it from the git history
- While we have it here it introduces additional mental complexity, as when changing code here we also have to think about this commented code